### PR TITLE
Deleting user

### PR DESCRIPTION
**Pull request by mosketrem**

### CODE REVIEW

File: app/controllers/user_controller.py [hunk 1]
```diff
@@ -7,8 +7,19 @@
 
 from app.models.User import User
```
Comment (0xporky) [inline app/controllers/user_controller.py]: Take a look on this [PR](https://github.com/Social-projects-Rivne/Cycling_admin/pull/3)
After it accepted you had to use  next construction to access to model
`from app.models.user import User`

File: app/models/User.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ def __init__(self, **kvargs):
         self.full_name = kvargs.get('full_name')
         self.email = kvargs.get('email')
         self.password = kvargs.get('password')
-        self.is_active = kvargs.get('is _active')
+        self.is_active = kvargs.get('is_active')
         self.avatar = kvargs.get('avatar')
         self.role_id = kvargs.get('role_id')
```
Comment (0xporky) [inline app/models/User.py]: Owh.. it may a coflict here, because i change something in user model. We'll resolve it may be in manual mode :)

**Comment by 0xporky** [comment]
@mosketrem Ohh, please merge your model with develop branch to resolve conflicts.

**Comment by mosketrem** [comment]
@0xporky нафіга ти назвав теку для тестів unitests ?

**Comment by psavyuk** [comment]
:( Guys, Did you read about __init__ in python? Probably, You have some extraordinary explanation which i need to hear? Please do not hesitate and write me))) MVC? and User is inconsistent name

**Comment by 0xporky** [comment]
@mosketrem it's just an type error. If you can, please rename it into 'unittests' and make another pull

**Pull request closed without merging**